app-creator/src/widgets/ui-map/ui-map.ts

```diff
@@ -37,106 +46,161 @@ function loadGoogleMaps(apiKey: string) {
 
 @customElement('ui-map')
 export class Map extends LitElement {
-  static styles = css`
-    #editable-view {
-      position: absolute;
-      top: 0;
-      right: 0;
-      display: flex;
-      justify-content: flex-end;
-      width: 100%;
-    }
-
-    .edit-buttons {
-      background-color: white;
-      border: var(--light-border);
-      border-radius: var(--extra-tight);
-      margin-left: var(--extra-tight);
-      cursor: pointer;
-    }
-  `;
+  static styles = css``;
 
   static attributes: AttributeMetaData = {
     // Default latitude and longitude is set to Google's Mountain View office.
     latitude: {
-      value: '39.930546488601294',
-      placeholder: '39.9305464',
+      value: '37.419857',
+      placeholder: '37.419857',
       step: 0.000000000000000000001,
       type: InputType.number,
     },
     longitude: {
-      value: '-100.825',
+      value: '-122.078827',
       step: 0.000000000000000000001,
-      placeholder: '-100.825',
+      placeholder: '-122.078827',
       type: InputType.number,
     },
     zoom: {
       value: '4',
       placeholder: '4',
       type: InputType.number,
     },
+    zoomControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    fullscreenControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    streetViewControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    mapTypeControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    mapStyles: {
+      value: 'standard',
+      type: InputType.select,
+      items: ['standard', 'silver', 'retro', 'night', 'dark', 'aubergine'],
+    },
+    customMapStyles: {
+      value: '',
+      placeholder: 'Past JSON here',
+      type: InputType.textarea,
+      tooltip: 'Create custom map styles at https://mapstyle.withgoogle.com/.',
```

Comment:
Maybe you can support a "Learn More" link which can be injected into the the paper-tooltip somehow? Otherwise, Joanna's idea for using a paper-dialog with `positionTarget`, styled to look like a tooltip would work pretty well! https://www.webcomponents.org/element/@polymer/paper-dialog/demo/demo/index.html

Reply:
![image](https://user-images.githubusercontent.com/26859947/85494621-05d17380-b5a7-11ea-8933-6513729d7dec.png)